pytest/bitops_test.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+import arkouda as ak
+import pytest
+
+A = B = EDGE_CASES = EDGE_CASES_UINT = None
+
+
+def set_globals():
+    global A, B, EDGE_CASES, EDGE_CASES_UINT
+
+    A = ak.arange(10)
+    B = ak.cast(A, ak.uint64)
+    EDGE_CASES = ak.array([-(2**63), -1, 2**63 - 1])
+    EDGE_CASES_UINT = ak.cast(ak.array([-(2**63), -1, 2**63 - 1]), ak.uint64)

```

Comment:
I am not sure we want to use globals here. I am thinking using a fixture may be better. Example below:

```python
@pytest.fixture
def bitops_data():
    return {
        'A': ak.arange(10),
        'B': ak.cast(A, ak.uint64),
        'EDGE_CASES': ak.array([-(2**63), -1, 2**63 - 1]),
        'EDGE_CASES_UINT': ak.cast(ak.array([-(2**63), -1, 2**63 - 1]), ak.uint64)
     }
```

Then in each test requiring the information you would pass `bitops_data` as a parameter and access each component like you would any other dictionary.

Reply:
Added `@pytest.fixture(scope="function", autouse=True)` 

Just `@pytest.fixture` was causing a lot of logging statements in the run, bloating the console output since the fixture was being ran on every test without the file